infra/Pos/Communication/Configuration.hs

```diff
@@ -8,18 +8,18 @@ module Pos.Communication.Configuration
 import           Data.Time.Units (Second)
 import           Universum
 
-import           Pos.Infra.Configuration (HasInfraConfiguration, ccMaxInvSize, ccMaxMempoolMsgSize,
-                                          ccMaxReqSize, ccNetworkWaitLogInterval,
+import           Pos.Infra.Configuration (HasInfraConfiguration, 
+                                          ccNetworkWaitLogInterval,
                                           infraConfiguration)
 
 networkWaitLogInterval :: HasInfraConfiguration => Second
 networkWaitLogInterval = fromIntegral . ccNetworkWaitLogInterval $ infraConfiguration
 
-maxReqSize :: HasInfraConfiguration => Word32
-maxReqSize = ccMaxReqSize infraConfiguration
+maxReqSize :: Word32
+maxReqSize = 102
```

Comment:
Yes I removed them from the record. The actual serialization format of the `ReqMsg` and `ResMsg` is not configurable, so why should their size limit be configurable? 61329f65d56d8693a38e8eef8ff649d54570dfd1

I can't see why this would be strictly necessary in this pull request. I'll look into it and put it in a dedicated patch if possible.

Reply:
You didn't remove the values from `configuration.yaml` also.